easybuild/easyconfigs/c/cpio/cpio-2.14-GCCcore-11.3.0.eb

```
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+name = 'cpio'
+version = '2.14'
+
+homepage = "https://savannah.gnu.org/projects/cpio/"
+description = """The cpio package contains tools for archiving."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://ftp.gnu.org/gnu/cpio/']
+sources = ['cpio-%(version)s.tar.bz2']
+checksums = ['fcdc15d60f7267a6fc7efcd6b9db7b6c8966c4f2fbbb964c24d41336fd3f2c12']
+
+builddependencies = [('binutils', '2.38')]
```

missing makeinfo builddep:
```suggestion
builddependencies = [
    ('binutils', '2.38'),
    ('makeinfo', '6.8'),
]
```